### ISSUE TITLE
Feedback on exercise 1

### ISSUE THREAD

**Issue by csarven** [zerzi-z/wbtc#1]
I'm not sure why you have two URI patterns, i.e., difference being: `/` and `#`. Can you justify it?

**Comment by zerzi-z** [comment]
Yes, some pages would get way to big (data wise and page "scroll" wise since there is a high possability of adding a lot of pictures and videos to some pages) e.g. the "http://pascal.zerzuben.name/interests". That’s why I decided to add the / to have the chance to create subtopic-pages instead of only using one page and the # in the process of creating the URI pattern.
I did use the # for education and work since I feel like those pages will not get to big.
I hope that this clarifies and justifies.

**Comment by csarven** [comment]
I suppose it would depend on whether you should have separate articles for them or not, sort of like blog posts. Consider the following: `/interests/paragliding#spots` vs `interests#paragliding-spots`, where the first would normally be a document about paragliding with fragment spots, and the second is a fragment on paragliding-spots. Sometimes it helps to consider what type of data would go under it.

Your justification is fine.. in a case like yours, I would consider whether any of these should be their own articles on the site instead of something going under strict classification. For instance, would 'interest' be a hierarchical classification on your site, or just a way to "tag" the thing about paragliding as an interest. Related to this is when your "interest" and your "work" aligns. If you are a paragliding instructor, where would that go?

**Comment by zerzi-z** [comment]
Yeah you are right considering what type of data goes under it will definitely help me for deciding on how to structure the site. During the process I had some kind of idea where what could go, but have not defined that clearly. 

I already answered this question in my exercise 1 github file ^^ because I had the same thought.
If you have a more preferable solution on how to "fix" this maybe occurring issue please let me know. (Because I am considering on constructing this site during my summer break for real)
Quote: "I know that there could be the possibility that a interest becomes a job and in this case there would be the opportunity to redirect the link within the page." Do you think this would be a solution that works? E.g. lets assume I become a paragliding instructor, paragliding is still my interest in addition now it is just also my work. On the interest page there is a link "paragliding instructor" which directs to "http://pascal.zerzuben.name/work#fluso". On the work page there would also be a link to the "spots where I fly" directing to "http://pascal.zerzuben.name/interests/paragliding#spots". 

Just for clarification "fluso" = the companies name of a paragliding school where I would work as a paragliding instructor. Since I was using the company names on the work page I wanted to be consistent for this example and chose the company name instead of "paragliding-instructor".

**Comment by csarven** [comment]
Well, this is all okay. You are currently getting to know the type of resources you will have on your site and how to refer to them. There is some trial and error in there. Remember the "Cool URI's don't change"? Keep that in mind. So, one way to go at it is to lean on a conservative approach on how you classify things. You can later break it down into more specific parts. For example, on my site, I use http://csarven.ca/cv as the document which lists what I want to share publicly with the world. It has fragments which point to different types of experiences, skills, events etc.. These are not blog posts or articles, but rather essentially a list of things.
